pkg/controller/knativeserving/knativeserving_controller.go

```diff
@@ -134,9 +134,6 @@ func (r *ReconcileKnativeServing) Reconcile(request reconcile.Request) (reconcil
 	instance := &servingv1alpha1.KnativeServing{}
 	if err := r.client.Get(context.TODO(), request.NamespacedName, instance); err != nil {
```

Comment:
Why do we need this condition at all anymore?

Reply:
Well yes, we load the instance here so we definitely need this call. On notfound we just want to abort, any other errors we want to surface.